devtools/maven/src/main/java/io/quarkus/maven/DevMojo.java

```diff
@@ -186,7 +186,8 @@ public void execute() throws MojoFailureException, MojoExecutionException {
         }
 
         if (!buildDir.isDirectory() || !new File(buildDir, "classes").isDirectory()) {
-            throw new MojoFailureException("The project has no output yet, run `mvn compile quarkus:dev`.");
+            throw new MojoFailureException("The project " + project.getName()
+                    + " has no output yet. Make sure it contains at least a source or resource file and then run `mvn compile quarkus:dev`.");
```

Comment:
I think, in this case, I would use `at least one` and not `at least a`.

Could you fix it?

Reply:
Done.